sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/specialized/BlobInputStream.java

```diff
@@ -81,21 +50,16 @@
      */
     private final BlobAccessConditions accessCondition;
 
-    /**
-     * Offset of the source blob this class is configured to stream from.
-     */
-    private final long blobRangeOffset;
 
     /**
      * Initializes a new instance of the BlobInputStream class.
      *
-     * @param blobClient A {@link BlobAsyncClientBase} object which represents the blob that this stream is associated
-     * with.
+     * @param blobClient A {@link BlobAsyncClient} object which represents the blob that this stream is associated with.
      * @param accessCondition An {@link BlobAccessConditions} object which represents the access conditions for the
      * blob.
      * @throws StorageException An exception representing any error which occurred during the operation.
      */
-    BlobInputStream(final BlobAsyncClientBase blobClient, final BlobAccessConditions accessCondition)
+    BlobInputStream(final BlobAsyncClient blobClient, final BlobAccessConditions accessCondition)
```

Comment:
I think this should be just BlobClient. Streams are used in the sync world, so it doesn't make sense to require a customer to pass in an async client, especially since we always block anyway.

Reply:
Change to sync will dramatically change the PR, which we can not fit the PR before Friday.
Async to Sync client change is an internal implementation change, so it is ok to do a post preview PR if it is needed.

Reply:
Can this feedback be addressed by adding a BlobsyncClient overload in a later update?

Reply:
Only private or protected method and constructor needs to update for this change. Our public APIs do not take client as param. We can simply change to sync instead of adding sync overload.